### ISSUE TITLE
After SSL enabled, network connection is lost

### ISSUE THREAD

**Issue by DrissiReda** [owncloud/ios#1002]
After I enabled SSL and started using HTTPS on my server, things are working fine on the desktop version. But on my ios app, I can establish connection to the server, then when I try uploading or downloading anything it says "network connection is lost.
HTTP to HTTPS is the only thing I changed, desktop apps work normally, but on ios or android apps I am unable to upload/download anything.

**Comment by jesmrec** [comment]
@DrissiReda could you send us a test account to apps@owncloud.com? also, if you can include your environment features (iOS version, device, server version, app version), it will be helpful to try to reproduce the problem.

**Comment by DrissiReda** [comment]
iPhone 5S iOS 9.3.3 jailbroken (but not many tweaks it's mainly for ssh)
Owncloud X 10.0.4 on an apache2 server.
iOS App version 3.7.2

**Comment by jesmrec** [comment]
do you have any proxy configuration in the middle?

**Comment by DrissiReda** [comment]
Nop, just a simple apache server

**Comment by DrissiReda** [comment]
@jesmrec any updates on this?

**Comment by jesmrec** [comment]
for the moment, it is not reproducible. If you can send a test account on your server, it would be helpful to reproduce it.

**Comment by DrissiReda** [comment]
@jesmrec I sent a test account to the email you have given me.

**Comment by jesmrec** [comment]
when did you send them? i checked our mailbox and didn't see. Please resend, and include a link to this issue in the message or subject. Thanks!

**Comment by DrissiReda** [comment]
@jesmrec there I sent it again, could you check the spam please? the sending email is "reda.drissi@yahoo.fr"

**Comment by jesmrec** [comment]
Thanks! i will give you feecback as soon as i check it

**Comment by jesmrec** [comment]
I have checked your URL and it is not reachable from here. Apart form an iOS9 device, i checked with Android and the web browser as well, with no success. Any problem with your server?

**Comment by DrissiReda** [comment]
@jesmrec sorry last night there was a power outtage, it is a home server so I got no UPS whatsoever, try again it should work perfectly I have used it this morning.

**Comment by jesmrec** [comment]
I have checked your server right now with an iOS9 (v3.7.2) and Android6 (v2.5.0) devices. Downloads worked and uploads as well. Did you perform any change in your server? All actions seems to work fine. Maybe are the tweaks for your jailbreak breaking something?

**Comment by DrissiReda** [comment]
that is strange... I'll try on different iPhones or emulators. Thank you !

**Comment by jesmrec** [comment]
but currently, does it work for you?

**Comment by DrissiReda** [comment]
it doesn't on iphone, browser version works, but on the iphone it always gives me "Network connection is lost" even if I reinstall the app, clear the cache remove the account and add it again.

**Comment by jesmrec** [comment]
also happens with android?

**Comment by DrissiReda** [comment]
I don't have an android, so can't tell

**Comment by jesmrec** [comment]
ok, as you mentioned in the first message that it does not wotk in both ios and android, i guessed that.

**Comment by DrissiReda** [comment]
yes I've tested it with my girlfriend's android few days ago, but I don't have an android device on hand right now so I can't test it.

**Comment by DrissiReda** [comment]
As I told you @jesmrec everything worked perfectly beffore SSL, once added I started having on and off issues( only for mobile), the app randomly freezes, and rarely works on my apache2 log ( name of the file is other_vhosts_access.log) I found this line : 
`host-001.home:443 37.168.224.23 - - [01/Mar/2018:15:30:02 +0100] "PROPFIND /owncloud/remote.php/webdav/InstantUpload/ HTTP/1.1" 403 232 "-"     "Mozilla/5.0 (iOS) ownCloud-iOS/3.7.2"`

But when it works I find this : 
host-001.home:443 37.168.224.23 - - [01/Mar/2018:15:30:02 +0100] "GET /owncloud/index.php/apps/files/api/v1/thumbnail/64/64/InstantUpload/P    hoto-2017-09-30-19-14-07_2768.JPG HTTP/1.1" 403 232 "-" "Mozilla/5.0 (iOS) ownCloud-iOS/3.7.2"

I hope this could help
